<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>967</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_prj">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>Prj</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_LoadRef">
        <property name="text">
         <string>Load Ref</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_RefFile">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>RefFile</string>
        </property>
        <property name="buddy">
         <cstring>pushButton_LoadRef</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_LoadTarget">
        <property name="text">
         <string>Load Target</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_TargetFile">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>TargetFile</string>
        </property>
        <property name="buddy">
         <cstring>pushButton_LoadTarget</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_calcDiff">
        <property name="text">
         <string>CalcDiff</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Select:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>Reference</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Target</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Diff</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTreeView" name="treeView_spreadSheets"/>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="7" column="0" colspan="3">
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="0">
             <widget class="QComboBox" name="comboBox_chartType">
              <item>
               <property name="text">
                <string>Table</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Bar</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Line</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Scatter</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Heatmap</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="comboBox_chartData">
              <item>
               <property name="text">
                <string>Ref</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Target</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Diff</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="pushButton_delChart">
              <property name="text">
               <string>DeleteChart</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="pushButton_addChart">
              <property name="text">
               <string>AddChart</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1" colspan="2">
             <widget class="QPushButton" name="pushButton_showChart">
              <property name="text">
               <string>ShowChart</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="3">
             <widget class="QTreeView" name="treeView_charts"/>
            </item>
           </layout>
          </item>
          <item row="4" column="2">
           <widget class="QPushButton" name="pushButton_delSheet">
            <property name="text">
             <string>DeleteSheet</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QLineEdit" name="lineEdit_sheetName">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="pushButton_addSheet">
            <property name="text">
             <string>AddSheet</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QPushButton" name="pushButton_showSheet">
            <property name="text">
             <string>Show</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QComboBox" name="comboBox_sheetType"/>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="comboBox_sheetGroup"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_newCounter">
          <property name="text">
           <string>New</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_selectCounters">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Names Filter</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_pfmonFilter</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_pfmonFilter">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Counter Expr</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_pfmonExpr</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_pfmonExpr"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Counter Name</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_pfmonNewName</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_pfmonNewName"/>
          </item>
         </layout>
        </item>
        <item row="3" column="0" colspan="2">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTreeView" name="treeView_pfmonSelected">
            <property name="selectionMode">
             <enum>QAbstractItemView::MultiSelection</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_pfmonSelectFilter"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_delSelectedCounters">
            <property name="text">
             <string>RemoveCounters</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QTreeView" name="treeView_pfmons">
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTreeView" name="treeView_batches">
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Select All</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton_selBatches">
            <property name="text">
             <string>AddBatchs</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLineEdit" name="lineEdit_batchFilter">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeView" name="treeView_batchSelected">
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_batchSelectedFilter"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_delBatches">
          <property name="text">
           <string>RemoveBatches</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>965</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <addaction name="menuOpen"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
